fe/fe-core/src/test/java/org/apache/doris/utframe/MockedBackendFactory.java

```diff
@@ -248,6 +263,62 @@ private void handleCloneTablet(TAgentTaskRequest request, TFinishTaskRequest fin
                     finishTaskRequest.setFinishTabletInfos(tabletInfos);
                 }
 
+                private void handleStorageMediumMigrate(TAgentTaskRequest request, TFinishTaskRequest finishTaskRequest) {
```

Comment:
just update src and dest disk's data used and avaialable capactity is ok,  like handleCloneTablet

Reply:
fixed